gp-includes/things/original.php

```diff
@@ -211,6 +211,10 @@ function( $entry ) {
 		foreach ( $translations->entries as $key => $entry ) {
 			$wpdb->queries = array();
 
+			if ( ! $entry->context ) {
+				$entry->context = '';
+			}
```

Comment:
This introduced a bug, see https://github.com/GlotPress/GlotPress/issues/1724. This should better be solved by checking for `is_string()` before calling `mb_strlen()`.